pages/question/question.css

```diff
@@ -39,3 +39,6 @@
 .wrong-option {
   background-color: var(--danger-color);
 }
+.rules-book a {
+  color: white;
```

Comment:
You can use the `--white` variable for this. You have used this variable in rules.css.

Reply:
I will change it!